pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -3701,46 +3540,37 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 
 	Context("with topologySpreadConstraints checks", func() {
 		var vmi *v1.VirtualMachineInstance
-		vmiAdmissionReviewFromTopologyConstraints := func(vmi *v1.VirtualMachineInstance, constraints []k8sv1.TopologySpreadConstraint) *admissionv1.AdmissionReview {
-			vmi.Spec.TopologySpreadConstraints = constraints
-			vmiBytes, _ := json.Marshal(&vmi)
-
-			return &admissionv1.AdmissionReview{
-				Request: &admissionv1.AdmissionRequest{
-					Resource: webhooks.VirtualMachineInstanceGroupVersionResource,
-					Object: runtime.RawExtension{
-						Raw: vmiBytes,
-					},
-				},
-			}
-		}
 		BeforeEach(func() {
 			vmi = newBaseVmi()
 			vmi.Spec.TopologySpreadConstraints = []k8sv1.TopologySpreadConstraint{}
```

Comment:
out of scope: drop this misleading assignment.
All the subtests are setting the `TopologySpreadConstraints` explicitly